util/gh-pages/index.html

```diff
@@ -511,6 +525,14 @@ <h1>Clippy Lints</h1>
                             </span>
                         </div>
                     </div>
+                    <div class="col-12 col-md-2 btn-group expansion-group">
+                        <button title="Collapse All" class="btn btn-default expansion-control" type="button" ng-click="toggleExpansion(data, false); $event.stopPropagation()">
```

Comment:
Can you explain why this needs to call `$event.stopPropagation()`? There are no other elements in the area which should affect this right?

Reply:
It does not. There are no parent event handlers that need to handle this event so I've just removed it.